### PR TITLE
Bugfix/optional label check radio

### PR DESCRIPTION
**Pull request by andrych**
* Fixed label not being optionally rendered for Checkbox and RadioButton

### CODE REVIEW

**Review by ian-emsens-shd — APPROVED** [review]

File: src/components/Checkbox/Checkbox.types.ts [hunk 1]
```diff
@@ -4,5 +4,5 @@ import { DefaultComponentProps, HTMLInputAttrs } from '../../types';
 
 export interface CheckboxProps extends DefaultComponentProps, HTMLInputAttrs {
 	checkIcon?: ReactNode;
-	label: string | ReactNode;
+	label?: string | ReactNode;
```
Comment (ian-emsens-shd) [inline src/components/Checkbox/Checkbox.types.ts:7]: Globaal even find & replace op `string | ReactNode` => `ReactNode`?

Comment (andrych) [reply src/components/Checkbox/Checkbox.types.ts:7]: Zal ik in een volgende PR fixen, string zit idd al mee in ReactNode 👍

File: src/components/RadioButton/RadioButton.types.ts [hunk 2]
```diff
@@ -3,5 +3,5 @@ import { ReactNode } from 'react';
 import { DefaultComponentProps, HTMLInputAttrs } from '../../types';
 
 export interface RadioButtonProps extends DefaultComponentProps, HTMLInputAttrs {
-	label: string | ReactNode;
+	label?: ReactNode;
```
Comment (bertyhell) [inline src/components/RadioButton/RadioButton.types.ts:6]: Why did you make this stricter?
aren't there use cases where string is also possible?

Comment (andrych) [reply src/components/RadioButton/RadioButton.types.ts:6]: ReactNode covers string types so it's still the same as before

**Pull request merged**